near-bindgen/src/collections/set.rs

```diff
@@ -39,7 +39,7 @@ where
     T: BorshSerialize + BorshDeserialize,
 {
     /// Serializes element into an array of bytes.
-    fn serialize_element(&self, element: T) -> Vec<u8> {
+    fn serialize_element(&self, element: &T) -> Vec<u8> {
```

Comment:
CC @mikhailOK since, he is using near-bindgen collections for EVM.

Reply:
yep, I'll update evm contract later. Currently it depends on the 0.2.5 version, so shouldn't be an issue